Parking-Zone/ViewModels/ParkingSlot/ParkingSlotEditVM.cs

```diff
@@ -28,6 +29,7 @@ public ParkingSlotEditVM(Models.ParkingSlot slot)
         [Required]
         public Guid ParkingZoneId { get; set; }
         public string? ParkingZoneName { get; set; }
+        public bool HasAnyActiveReservation { get; set; } = false;
```

Comment:
```suggestion
        public bool HasAnyActiveReservation { get; set; };
```
Boolean property is false by default.